exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,38 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number <= 0) {
+    return true;
+  }
+
+  return false;
+}
+
+function isBetween5and10(num) {
+  if (num >= 5 && num <= 10) {
+    return true;
+  }
+
+  return false;
+}
```

Comment:
Any function that has simple check like this one, you can just return the comparison instead like so:

```
function isBetween5and10(num) {
  return num >= 5 && num <= 10;
}
```